docs/style-guide.rst

```diff
@@ -1,10 +1,45 @@
+-----------
 Style Guide
-===========
+-----------
+
+This document specifies how we write on our websites.
+It's just a start,
+but everything has to start somewhere :)
+
+Part 1: Technical Formatting
```

Comment:
There is no reason for these to be manually numbered.

Reply:
Unless I'm missing something?

Reply:
I think mostly to be able to reference them, but probably not.

Reply:
Yeah, numbering was for easy referencing. The Chicago Manual of Style uses it, and it's one of features I like about it. But it might not be as useful for style guide that's this small.